app/helpers/display_format_helper.rb

```diff
@@ -5,7 +5,15 @@ def date_format(date)
   end
 
   def head_info_list_maker(info)
-    [info['format'].humanize, date_format(info['public_timestamp'])]
+    head_info_list = [info['format'].humanize, date_format(info['public_timestamp'])]
```

Comment:
I feel like this might be nicer being implemented as a class that wraps the info object rather than a helper.